ui/css/logviewer.css

```diff
@@ -77,10 +77,29 @@ body {
     white-space: pre;
 }
 
+.lv-line-num {
+    white-space: pre;
+    float:left;
+    align-items: center;
+    border-right-style: solid;
+    border-right-color:  #F8EEC7;
+    border-right-width: thin;
+    -moz-user-select: none;
+    -khtml-user-select:none;
+    -webkit-user-select:none;
+    -o-user-select :none;
+    background-color: white;
+}
```

Comment:
A minor nit, but in all cases in the PR you can use this convention below consistency, a single character of white space between the property and the value.

`property: value;`